edxval/models.py

```diff
@@ -199,12 +201,45 @@ def deconstruct(self):
         return name, path, args, kwargs
 
 
+class ListField(models.CharField):
+    """
+    ListField use to store and retrieve list data.
+    """
+    __metaclass__ = models.SubfieldBase
+
+    def get_prep_value(self, value):
+        """
+        Converts a list to its json represetation to store in database as text.
+        """
+        return json.dumps(value)
+
+    def to_python(self, value):
+        """
+        Converts the value into a list.
+        """
+        if not value:
+            value = []
+
+        if isinstance(value, list):
+            return value
+
+        try:
+            py_list = json.loads(value)
+            if not isinstance(py_list, list):
+                raise TypeError
+        except (ValueError, TypeError):
+            raise ValidationError(u'Must be a valid list.')
+
+        return py_list
+
+
 class VideoImage(TimeStampedModel):
     """
     Image model for course video.
     """
     course_video = models.OneToOneField(CourseVideo, related_name="video_image")
     image = CustomizableImageField()
+    urls = ListField(max_length=2000)
```

Comment:
no it can not. our urls looks like `video-images/390326dfb3b24486b23f6dd1ed47e997.jpg`